lib/internal/http2/compat.js

```diff
@@ -71,19 +84,24 @@ function statusMessageWarn() {
 }
 
 function onStreamData(chunk) {
-  if (!this[kRequest].push(chunk))
+  const request = this[kRequest];
+  if (request !== undefined && !request.push(chunk))
```

Comment:
Should `request === undefined` even be possible here?

Reply:
Technically? Yes. Likely? No.